### PR TITLE
feat(update):2024-11-25 Conflict Update

### PR DESCRIPTION
**Pull request by Zitchas**
There's a conflict...

### CODE REVIEW

**Comment by TheGiraffe3** [comment]
What is it? The Arm CI build?

**Comment by Zitchas** [comment]
The conflict in question:

L121/L122 of ShipInfoPanel.cpp

```
	// Draw the interface.
	const Interface *infoPanelUi = GameData::Interfaces().Get("info panel");
<<<<<<< f(u)20241125conflictupdate
	infoPanelUi->Draw(info, this);
=======
	infoPanelUi->Draw(interfaceInfo, this);
	int infoPanelLine = 0;
>>>>>>> experimental
```

Went with the one already in experimental, but I suspect this was wrong.

I smell code churn:

A whole bunch of `interfaceInfo.` became `info`, while a bunch of things that already were `info.` became a longer `infosomething.` (memory failed me as to exactly what it was.) Don't see the point right off the bat, but I'll have to take a look at it later before merging this. (and probably tweaking the line, but I'm not sure yet)

File: source/ShipInfoPanel.cpp [hunk 1]
```diff
@@ -118,8 +118,8 @@ void ShipInfoPanel::Draw()
 
 	// Draw the interface.
 	const Interface *infoPanelUi = GameData::Interfaces().Get("info panel");
-	infoPanelUi->Draw(interfaceInfo, this);
-	int infoPanelLine = 0;
+	infoPanelUi->Draw(info, this);
+	// int infoPanelLine = 0; Removed by the 20241125conflictupdate not sure if it is upstream or Delta's.
```
Comment (TheGiraffe3) [inline source/ShipInfoPanel.cpp]:
```suggestion
	// int infoPanelLine = 0; Removed by the 2024-11-25 conflict update. Not sure if it is upstream or Delta's.
```

Comment (Zitchas) [reply source/ShipInfoPanel.cpp]: That word is directly referencing the name of my branch title, so I would prefer to keep it as the branch title and not spacing it out as a sentence. It'll also be deleted before this is merged.

Good spotting, though. :)

Comment (TheGiraffe3) [reply source/ShipInfoPanel.cpp]: If it'll be deleted then feel free to ignore. Just pointing it out.

**Comment by Zitchas** [comment]
Alrighty, so that dropped this particular update from 50 files to 31 files changed. lol. :)

**Comment by Zitchas** [comment]
Whole bunch of extra updating, predominantly to the HardpointInfoPanel.cpp file, but some in other info panel related places to get it all working again.

No tooltips show up in the info panels. DId there used to be? I think there used to be, but not sure. Unfortunately can't test right now. There's no tooltips showing up now.

**Comment by Zitchas** [comment]
OK, last thing to track down and resolve: Tooltips not showing on either info panel.

**Comment by TheGiraffe3** [comment]
Looks like the conflict may be dealt with by revertation. If the tooltips worked before, it's probably fine to close this; the conflicting commit is going to be reverted.

**Comment by Zitchas** [comment]
The conflict itself has been fixed. It was fairly straightforward. The tooltip thing, now that I am testing it, seems to not work back through several releases.

**Pull request merged**